packages/docprovider-extension/src/filebrowser.ts

```diff
@@ -45,6 +45,11 @@ namespace CommandIDs {
 }
 const DOCUMENT_TIMELINE_URL = 'api/collaboration/timeline';
 
+const TWO_SESSIONS_WARNING =
+  'You have opened the file %1 with two separate views. ' +
+  'This is not supported. Please close one view; otherwise, ' +
+  'some of your edits may not be saved properly.';
```

Comment:
It's still not completely true to me. When saying "You have opened..." it seems to mean that the same user has opened the two views, but this user may have opened only one view. Also, we could be more specific an say "Please close this view" instead of "Please close one view".
```suggestion
  'The file %1 has been opened with two different views. ' +
  'This is not supported. Please close this view; otherwise, ' +
  'some of your edits may not be saved properly.';
```